examples/bert/data/download_glue_data.py

```diff
@@ -129,19 +129,21 @@ def main(arguments) -> None:
     for task in tasks:
         if task == 'MRPC':
             import subprocess
-            if not os.path.exists("data/MRPC"):
-                subprocess.run("mkdir data/MRPC", shell=True)
             # pylint: disable=line-too-long
+            if not os.path.exists("data/MRPC"):
+                subprocess.run("mkdir data/MRPC", shell=True, check=False)
             subprocess.run(
                 'wget -P data/MRPC/ https://dl.fbaipublicfiles.com/senteval/senteval_data/msr_paraphrase_train.txt',
-                shell=True)
+                shell=True, check=False)
             subprocess.run(
                 'wget -P data/MRPC/ https://dl.fbaipublicfiles.com/senteval/senteval_data/msr_paraphrase_test.txt',
-                shell=True)
+                shell=True, check=False)
```

Comment:
Do we really want `check=False`? We'll be needing these files, so a non-zero return code would mean that the file is not downloaded, which would cause problems. To me it's better if such error is not ignored.